cli/dcoscli/package/main.py

```diff
@@ -5,7 +5,8 @@
     dcos package --info
     dcos package describe [--app --options=<file> --cli] <package_name>
     dcos package install [--cli | [--app --app-id=<app_id>]]
-                         [--options=<file> --yes] <package_name>
```

Comment:
We need to standardize on whether we group all optional args into a single pair of brackets, or wrap each one independently.  I like the latter approach, which means we should split up [--options] and [--yes] here